app/mailers/member_mailer.rb

```diff
@@ -2,6 +2,7 @@ class MemberMailer < ApplicationMailer
   def notify_bid
     @name = params[:name]
     @bid = params[:bid]
+    @membership_id = params[:membership_id]
     mail(to: params[:email], subject: 'Dein Mitgliedsbeitrag wurde erfolgreich gespeichert')
```

Comment:
Here should be "Dein Gebot wurde erfolgreich gespeichert"

Reply:
good point :-)